README.md

```diff
@@ -203,27 +203,22 @@ Refer to the [official docs](https://eslint.org/docs/latest/user-guide/configuri
 The schema of the `settings.react` object would be identical to that of what's already described above in the legacy config section.
 
 <!-- markdownlint-disable-next-line no-duplicate-heading -->
-### Shareable configs
-
-There're also 3 shareable configs.
-
-- `eslint-plugin-react/configs/all`
-- `eslint-plugin-react/configs/recommended`
-- `eslint-plugin-react/configs/jsx-runtime`
+### Flat Configs
 
```

Comment:
it seems like we need an "eslintrc Configs" or "Legacy Configs" section, that preserves the non-flat examples?

Reply:
I believe the section for eslintrc configs is still in the document:

https://github.com/jsx-eslint/eslint-plugin-react/blob/eb4d749f0236b8e7a80d15755701426206085ea9/README.md#shareable-configs

Reply:
aha, thanks, i must have missed it in the diff view.